docs/_book.yaml

```diff
@@ -0,0 +1,86 @@
+upper_tabs:
```

Comment:
Do we need a copyright notice in this file?

Reply:
yes, adding